### ISSUE TITLE
Assets are missing in the dev environment

### ISSUE THREAD

**Issue by oyiptong** [oyiptong/splice#242]
Here's an error I get in the console:

```
downloadable font: download failed (font-family: "FontAwesome" style:normal weight:normal stretch:normal src index:1): status=2147746065 source: http://splice_tiles.dev.mozaws.net/campaign/public/fonts/fontawesome-webfont.woff2 campaign:4:14
downloadable font: download failed (font-family: "FontAwesome" style:normal weight:normal stretch:normal src index:2): status=2147746065 source: http://splice_tiles.dev.mozaws.net/campaign/public/fonts/fontawesome-webfont.woff campaign:4:14
downloadable font: download failed (font-family: "FontAwesome" style:normal weight:normal stretch:normal src index:3): status=2147746065 source: http://splice_tiles.dev.mozaws.net/campaign/public/fonts/fontawesome-webfont.ttf
```

**Comment by oyiptong** [comment]
Had a screenshare session with @tkiethanom. We discovered that the assets aren't being packaged/deployed correctly. Many files are missing:

Observed:
```
[oyiptong@ip-172-31-34-44 static]$ pwd
/opt/splice/lib/python2.7/site-packages/splice/static
[oyiptong@ip-172-31-34-44 static]$ find .
.
./build
./build/README.txt
./build/css
./build/css/app.css
./build/css/datetime.css
./build/css/newtab.css
./build/img
./build/img/ajax-loader.gif
./build/js
./build/js/bundle.js
```

Expected:
```
(splice-env)oyiptong@quanta:~/Projects/up-projects/splice/build/lib/splice/static/build/campaign-manager$ find .
.
./dist
./dist/index.html
./dist/main.js
./dist/public
./dist/public/img
./dist/public/img/ajax-loader-aqua.gif
./dist/public/img/ajax-loader-dark.gif
./dist/public/img/ajax-loader-navy.gif
./dist/public/img/ajax-loader.gif
./dist/public/img/breadcrumbs
./dist/public/img/breadcrumbs/breadcrumb-arrow-active.png
./dist/public/img/breadcrumbs/breadcrumb-arrow.png
./dist/public/img/breadcrumbs/breadcrumb-branch-short-active.png
./dist/public/img/breadcrumbs/breadcrumb-branch-short.png
./dist/public/img/breadcrumbs/breadcrumb-line-active.png
./dist/public/img/breadcrumbs/breadcrumb-line.png
./dist/public/img/custom-radio.png
./dist/public/img/favicon.ico
./dist/public/img/select-downarrow-dark.png
./dist/public/img/select-downarrow.png
./dist/public/img/select-uparrow-dark.png
./dist/public/img/select-uparrow.png
./dist/public/img/timelines
./dist/public/img/timelines/dot-active.png
./dist/public/img/timelines/dot.png
./dist/public/img/timelines/line-active.png
./dist/public/img/timelines/line.png
./dist/public/mock
./dist/public/mock/account.json
./dist/public/mock/account_1.json
./dist/public/mock/account_2.json
./dist/public/mock/account_3.json
./dist/public/mock/accounts.json
./index.html
./main.js
./public
./public/img
./public/img/ajax-loader-aqua.gif
./public/img/ajax-loader-dark.gif
./public/img/ajax-loader-navy.gif
./public/img/ajax-loader.gif
./public/img/breadcrumbs
./public/img/breadcrumbs/breadcrumb-arrow-active.png
./public/img/breadcrumbs/breadcrumb-arrow.png
./public/img/breadcrumbs/breadcrumb-branch-short-active.png
./public/img/breadcrumbs/breadcrumb-branch-short.png
./public/img/breadcrumbs/breadcrumb-line-active.png
./public/img/breadcrumbs/breadcrumb-line.png
./public/img/custom-radio.png
./public/img/favicon.ico
./public/img/select-downarrow-dark.png
./public/img/select-downarrow.png
./public/img/select-uparrow-dark.png
./public/img/select-uparrow.png
./public/img/timelines
./public/img/timelines/dot-active.png
./public/img/timelines/dot.png
./public/img/timelines/line-active.png
./public/img/timelines/line.png
./public/mock
./public/mock/account.json
./public/mock/account_1.json
./public/mock/account_2.json
./public/mock/account_3.json
./public/mock/accounts.json
```

**Comment by oyiptong** [comment]
observed is from the wrong machine, that was staging. trying dev now

**Comment by oyiptong** [comment]
[oyiptong@ip-172-31-43-219 static]$ pwd
/opt/splice/lib/python2.7/site-packages/splice/static
[oyiptong@ip-172-31-43-219 static]$ find .
.
./.gitkeep
./build
./build/campaign-manager
./build/campaign-manager/index.html
./build/campaign-manager/main.js
./build/campaign-manager/public
./build/campaign-manager/public/fonts
./build/campaign-manager/public/fonts/fontawesome-webfont.eot
./build/campaign-manager/public/fonts/fontawesome-webfont.svg
./build/campaign-manager/public/fonts/fontawesome-webfont.ttf
./build/campaign-manager/public/fonts/fontawesome-webfont.woff
./build/campaign-manager/public/fonts/fontawesome-webfont.woff2
./build/campaign-manager/public/fonts/glyphicons-halflings-regular.eot
./build/campaign-manager/public/fonts/glyphicons-halflings-regular.svg
./build/campaign-manager/public/fonts/glyphicons-halflings-regular.ttf
./build/campaign-manager/public/fonts/glyphicons-halflings-regular.woff
./build/campaign-manager/public/fonts/glyphicons-halflings-regular.woff2
./build/campaign-manager/public/img
./build/campaign-manager/public/img/ajax-loader-aqua.gif
./build/campaign-manager/public/img/ajax-loader-dark.gif
./build/campaign-manager/public/img/ajax-loader-navy.gif
./build/campaign-manager/public/img/ajax-loader.gif
./build/campaign-manager/public/img/alpha-horizontal.png
./build/campaign-manager/public/img/alpha.png
./build/campaign-manager/public/img/breadcrumb-arrow-active.png
./build/campaign-manager/public/img/breadcrumb-arrow.png
./build/campaign-manager/public/img/breadcrumb-branch-short-active.png
./build/campaign-manager/public/img/breadcrumb-branch-short.png
./build/campaign-manager/public/img/breadcrumb-line-active.png
./build/campaign-manager/public/img/breadcrumb-line.png
./build/campaign-manager/public/img/breadcrumbs
./build/campaign-manager/public/img/breadcrumbs/breadcrumb-arrow-active.png
./build/campaign-manager/public/img/breadcrumbs/breadcrumb-arrow.png
./build/campaign-manager/public/img/breadcrumbs/breadcrumb-branch-short-active.png
./build/campaign-manager/public/img/breadcrumbs/breadcrumb-branch-short.png
./build/campaign-manager/public/img/breadcrumbs/breadcrumb-line-active.png
./build/campaign-manager/public/img/breadcrumbs/breadcrumb-line.png
./build/campaign-manager/public/img/custom-radio.png
./build/campaign-manager/public/img/dot-active.png
./build/campaign-manager/public/img/dot.png
./build/campaign-manager/public/img/favicon.ico
./build/campaign-manager/public/img/hue-horizontal.png
./build/campaign-manager/public/img/hue.png
./build/campaign-manager/public/img/line-active.png
./build/campaign-manager/public/img/line.png
./build/campaign-manager/public/img/saturation.png
./build/campaign-manager/public/img/select-downarrow-dark.png
./build/campaign-manager/public/img/select-downarrow.png
./build/campaign-manager/public/img/select-uparrow-dark.png
./build/campaign-manager/public/img/select-uparrow.png
./build/campaign-manager/public/img/timelines
./build/campaign-manager/public/img/timelines/dot-active.png
./build/campaign-manager/public/img/timelines/dot.png
./build/campaign-manager/public/img/timelines/line-active.png
./build/campaign-manager/public/img/timelines/line.png
./build/campaign-manager/public/mock
./build/campaign-manager/public/mock/account.json
./build/campaign-manager/public/mock/account_1.json
./build/campaign-manager/public/mock/account_2.json
./build/campaign-manager/public/mock/account_3.json
./build/campaign-manager/public/mock/accounts.json
./build/distribution-viewer
./build/distribution-viewer/css
./build/distribution-viewer/css/app.css
./build/distribution-viewer/css/newtab.css
./build/distribution-viewer/js
./build/distribution-viewer/js/bundle.js

**Comment by oyiptong** [comment]
So that looks more correct.

**Comment by oyiptong** [comment]
So, the issue is the deployed URL:

for instance, the app expects:
http://splice_tiles.dev.mozaws.net/campaign/public/img/timelines/line.png

But nginx serves at:
http://splice_tiles.dev.mozaws.net/static/build/campaign-manager/public/img/timelines/line.png

**Comment by rlr** [comment]
This should be fixed now. But the latest code isn't on mozilla/sp2 and right now I can't seem to push cleanly to it.

**Comment by rlr** [comment]
#263 should fix the remaining issues.

**Comment by k88hudson** [comment]
Resolved by 500dd01c5cbdff16533579c05354775cdf43ae16
